<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GRadioButtonView</class>
 <widget class="QWidget" name="GRadioButtonView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GRadioButtonView</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="m_title2">
       <property name="text">
        <string>Male or female?</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="m_male">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Male</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="m_female">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Female</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="m_notSure">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Not sure</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="m_title">
       <property name="text">
        <string>Are you ready?</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="m_yes">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Yes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="m_no">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>No</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="m_maybe">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Maybe</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_run">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>250</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
  <zorder>radioButton</zorder>
  <zorder>m_yes</zorder>
  <zorder>m_no</zorder>
  <zorder>m_maybe</zorder>
  <zorder>m_title</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>m_title2</zorder>
  <zorder>verticalSpacer</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
